### PR TITLE
[11.0][ADD] stock_inventory_virtual_location

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Stock Inventory Virtual Location
================================

This module extends the inventory adjustment by allowing to change
the virtual location.

Commented in https://github.com/OCA/stock-logistics-warehouse/issues/714.

### CODE REVIEW

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (JordiBForgeFlow) [inline stock_inventory_virtual_location/models/stock_inventory_line.py]: Please add a hook here...

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]

File: stock_inventory_lockdown/models/stock_move.py
Comment (rousseldenis) [inline stock_inventory_lockdown/models/stock_move.py]: I think this should not be in this PR

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (MiquelRForgeFlow) [reply stock_inventory_virtual_location/models/stock_inventory_line.py]: I think its not needed. There isn't any other customization of this method in OCA.

File: stock_inventory_virtual_location/__manifest__.py
Comment (rousseldenis) [inline stock_inventory_virtual_location/__manifest__.py:4]: Do you plan to add missing entries here (maintainers, dev.._status, ...) ?

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (rousseldenis) [inline stock_inventory_virtual_location/models/stock_inventory_line.py]: Why not using float_compare here ?

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (rousseldenis) [inline stock_inventory_virtual_location/models/stock_inventory_line.py]: I would have added 'move_obj = self.env['stock.move'] before the loop

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Code review

File: stock_inventory_lockdown/models/stock_move.py
Comment (MiquelRForgeFlow) [reply stock_inventory_lockdown/models/stock_move.py]: Is in https://github.com/OCA/stock-logistics-warehouse/pull/716, but I put here also because I wanted a green travis-ci!

File: stock_inventory_virtual_location/__manifest__.py
Comment (MiquelRForgeFlow) [reply stock_inventory_virtual_location/__manifest__.py:4]: I don't know. I haven't thought about this yet.

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (MiquelRForgeFlow) [reply stock_inventory_virtual_location/models/stock_inventory_line.py]: because odoo also doesn't use float_compare in their original method 😂

Comment (MiquelRForgeFlow) [reply stock_inventory_virtual_location/models/stock_inventory_line.py]: I will add

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (MiquelRForgeFlow) [reply stock_inventory_virtual_location/models/stock_inventory_line.py]: ok

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (rousseldenis) [reply stock_inventory_virtual_location/models/stock_inventory_line.py]: Higher quality in OCA :smile:

File: stock_inventory_lockdown/models/stock_move.py
Comment (rousseldenis) [reply stock_inventory_lockdown/models/stock_move.py]: Ok, in PR description, put 'Includes #716' instead of depends

Comment (MiquelRForgeFlow) [reply stock_inventory_lockdown/models/stock_move.py]: Well, my idea is wait until the other Pr is merged, and then rebase this one.

Comment (rousseldenis) [reply stock_inventory_lockdown/models/stock_move.py]: Yeah I've understood that. My remark just to help reviewer :smiley:

**Comment by mreficent** [comment]
Working now in the `stock_inventory_virtual_location_change_qty_reason` glue module 😉

**Comment by JordiBForgeFlow** [comment]
cc @nuriamartinxifre

**Comment by MiquelRForgeFlow** [comment]
Everybody, this PR is ready to review! :)

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@jbeficent Could you update your review ?

**Review by PhatDang — COMMENTED** [review]
LGTM, thanks!

**Review by NuriaXifre — COMMENTED** [review]
Functional review done!

**Review by NuriaXifre — APPROVED** [review]
Functional review done!

File: stock_inventory_virtual_location/models/stock_inventory_line.py
Comment (JordiBForgeFlow) [inline stock_inventory_virtual_location/models/stock_inventory_line.py]: @mreficent  please default to blank

Comment (JordiBForgeFlow) [reply stock_inventory_virtual_location/models/stock_inventory_line.py]: Well, do not default at all

Comment (MiquelRForgeFlow) [reply stock_inventory_virtual_location/models/stock_inventory_line.py]: done

**Review by LoisRForgeFlow — APPROVED** [review]
Functional review, LGTM

**Comment by rousseldenis** [comment]
@jbeficent Could you update your review ?

**Pull request merged**
